asctec_msgs/GPSData gpsdata
---
geometry_msgs/PoseStamped pose
